src/customHooks/useLeaderBoard/useLeaderBoard.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from "@tanstack/react-query";
+import { getLeaderBoardFromLocalStorage } from "./localStorageHandler";
+import { loadingMessage } from "./strings";
+import { errorMessage } from "./utils";
+
+export function useLeaderBoard() { 
+    const { isLoading, isError, error, data } = useQuery(['leaderBoard'], () => getLeaderBoardFromLocalStorage());   

```

Comment:
Use unique ids.
You don't need to wrap getLeaderBoardFromLocalStorage with an arrow function.